src/test/system/systest_runnable.cc

```diff
@@ -176,7 +177,7 @@ void SysTestRunnable::
 update_id_str(bool started)
 {
   bool use_threads = SysTestSettings::inst().use_threads();
-  char extra[128];
+  char extra[std::numeric_limits<int>::digits10 + 1];
```

Comment:
Ok, numeric_limits<>::digits10 is constexpr. I don't think the addition will change that. (The danger otherwise is that we might unintentionally trigger VLA behavior.)

Reply:
Yes to both, agreed.